config/initializers/inflections.rb

```diff
@@ -14,3 +14,6 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'TT' # Teacher Training
```

Comment:
Huh, cool - haven't done this before. 

If I understand correctly, this'll help Rails along when it's parsing the `TTApplication` name into various magic folder/file names?

Reply:
Yes — if you don't do this, it will expect the model in `tt_application.rb` to be `TtApplication`, which is, like, totally suboptimal